### ISSUE TITLE
Doc use a deperecated sphinx module

### ISSUE THREAD

**Issue by alarig** [RIPE-NCC/ripe-atlas-tools#219]
Hi,

While building doc, I have this error
```
Running Sphinx v2.4.4

Extension error:
Could not import extension sphinx.ext.pngmath (exception: No module named 'sphinx.ext.pngmath')
```

And this issue at sphinx says that this module shouldn’t be used anymore: https://github.com/sphinx-doc/sphinx/issues/6182#issuecomment-473617088

**Comment by EnterChaser** [comment]
https://github.com/sphinx-doc/sphinx/issues/6182#issuecomment-473617088 </

**Comment by EnterChaser** [comment]
> https://github.com/sphinx-doc/sphinx/issues/6182#issuecomment-473617088 </

**Comment by EnterChaser** [comment]
/<

**Comment by EnterChaser** [comment]
> Hi,
> 
> 
> 
> While building doc, I have this error
> 
> ```
> 
> Running Sphinx v2.4.4
> 
> 
> 
> Extension error:
> 
> Could not import extension sphinx.ext.pngmath (exception: No module named 'sphinx.ext.pngmath')
> 
> ```
> 
> 
> 
> And this issue at sphinx says that this module shouldn’t be used anymore: https://github.com/sphinx-doc/sphinx/issues/6182#issuecomment-473617088

**Comment by chrisamin** [comment]
Fixed in v3.0.0
